internal/api/v1/application/logs.go

```diff
@@ -190,3 +192,20 @@ func (hc Controller) streamPodLogs(ctx context.Context, conn *websocket.Conn, na
 
 	return conn.Close()
 }
+
+// https://pkg.go.dev/github.com/gorilla/websocket#hdr-Origin_Considerations
+// Regarding matching accessControlAllowOrigin and origin header:
+// https: //developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
+func newUpgrader() websocket.Upgrader {
+	return websocket.Upgrader{
+		CheckOrigin: func(r *http.Request) bool {
+			accessControlAllowOrigin := strings.TrimSuffix(viper.GetString("access-control-allow-origin"), "/")
+			originHeader := r.Header.Get("Origin")
+
+			return accessControlAllowOrigin == "" ||
```

Comment:
What if "viper.GetString("access-control-allow-origin" returns "developer.mozilla.org
" instead of "https://developer.mozilla.org"

Do I have a wait to identify it?

Reply:
Reading the docs, I'm not even sure if that a correct thing to do. Having said that, I think this feature is only for our UI in which case we get to decide what we set it to. As long as we set it to something that works no the server side, we are good. I'm not sure it's worth spending too much time to catch all possible cases.